include/cuse_lowlevel.h

```diff
@@ -9,8 +9,8 @@
   Read example/cusexmp.c for usages.
 */
 
-#ifndef _CUSE_LOWLEVEL_H_
-#define _CUSE_LOWLEVEL_H_
+#ifndef CUSE_LOWLEVEL_H_
```

Comment:
@pramodk-git, the trailing ``_`` is a stylistic one not mandated by the standard. I am happy to remove them if the project would prefer it to be done that way before the merge, but that is not a decision I feel comfortable dictating since it is a stylistic one.